src/components/application_manager/src/commands/hmi/on_exit_application_notification.cc

```diff
@@ -62,17 +62,21 @@ void OnExitApplicationNotification::Run() {
     return;
   }
 
+  HMILevel::eType app_hmi_level =
+      application_manager_.GetDefaultHmiLevel(app_impl);
   Common_ApplicationExitReason::eType reason;
```

Comment:
@dcherniev I don't understand logic of this fix. Why did you use default hmi level?